tutorials/plot_visualize_evoked.py

```diff
@@ -137,6 +137,9 @@
 # with :func:`mne.viz.plot_compare_evokeds`.
 # The plot is styled with dictionary arguments, again using "/"-separated tags.
 # We plot a MEG channel with a strong auditory response.
+#
+# For move advanced plotting using :func:`mne.viz.plot_compare_evokeds`
+# see also :ref:`tut_metadata_epochs`.
```

Comment:
This too - I didn't put it here.

Should the line be changed? @larsoner

Reply:
@agramfort did:

https://github.com/mne-tools/mne-python/pull/4526/commits/cfb5ce6425b9bc368786fa9829bfef35ebad039c

It's fine to leave the new sentences but remove the `_tut_` ref and replace it with the standard one. You can check correctness locally with `cd doc/; make html_dev-noplot`, you should only see 5 (I think?) warnings about gallery images missing in `manual/`.

Reply:
Oh ok.